platform/broadcom/sonic-platform-modules-dell/s6100/sonic_platform/module.py

```diff
@@ -198,3 +198,49 @@ def get_system_eeprom_info(self):
                   ‘0x26’:’01’, ‘0x27’:’REV01’, ‘0x28’:’AG9064-C2358-16G’}
         """
         return self._eeprom.system_eeprom_info()
+
```

Comment:
@aravindmani-1 can you run the sonic-mgmt test without these changes? These test are supposed to run only for T2 chasssis.Ref:-https://github.com/sonic-net/sonic-mgmt/blob/master/tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml#L317

Reply:
Without these changes, platform cases in module.py fails.